boilerplate/openshift/golang-osd-operator/standard.mk

```diff
@@ -125,3 +115,44 @@ yaml-validate: python-venv
 .PHONY: validate-olm-deploy-yaml
 validate-olm-deploy-yaml: python-venv
 	${PYTHON} ${CONVENTION_DIR}/validate-yaml.py $(shell git ls-files 'deploy/*.yaml' 'deploy/*.yml')
+
+######################
+# Targets used by prow
+######################
+
+# validate: Ensure code generation has not been forgotten; and ensure
+# generated and boilerplate code has not been modified.
+# TODO
+.PHONY: validate
+validate: ;
+
+# lint: Perform static analysis.
+.PHONY: lint
+lint: validate-olm-deploy-yaml gocheck
+
+# test: "Local" unit and functional testing.
+.PHONY: test
+test: gotest
+
+# coverage: Code coverage analysis and reporting.
+.PHONY: coverage
+coverage:
+	${CONVENTION_DIR}/codecov.sh
+
+# build: Code compilation and bundle generation. This should do as much
+# of what app-sre does as possible, so that there are no surprises after
+# a PR is merged.
+# TODO: Include generating (but not pushing) the bundle
+.PHONY: build
+build: docker-build
+
+#########################
+# Targets used by app-sre
+#########################
+
+# build-push: Construct, tag, and push the official operator and
+# registry container images.
+# TODO: Boilerplate this script.
+.PHONY: build-push
+build-push:
+	hack/app_sre_build_deploy.sh
```

Comment:
This needs to be the path under `${CONVENTION_DIR}` not the consumer repo's hack directory.

Reply:
Not yet. A separate PR will boilerplate the script (per the TODO) -- but it'll have to do it in such a way that it _still_ calls out to the repository's bespoke bundle generation. And then a PR after _that_ will figure out how to do that bundle gen generically from within boilerplate.

All this is trying to accomplish right now is allow us to change lines like [this one](https://gitlab.cee.redhat.com/service/app-interface/-/blob/030a6acfc5c614796c7a64010ef34d75a9365f5f/data/services/osd-operators/cicd/ci-int/jobs-aws-account-operator.yaml#L24) to say `make build-push` instead. Then we'll be able to swap out what's underneath that without having to keep changing app-interface.